motor/inventory/manager.go

```diff
@@ -65,7 +68,7 @@ func WithVault(v vault.Vault) Option {
 
 func WithCachedCredsResolver() Option {
 	return func(im *inventoryManager) error {
-		im.CredsResolver = credentials_resolver.New(im.GetVault(), false)
```

Comment:
This branch has enableCaching always false. Not sure what's going on

Reply:
That does not make any sense. But we should find occurrences of WithCachedCredsResolver and remove them for now

Reply:
and set isCached to true here